tests/bootstrap_project/test/project_test.dart

```diff
@@ -171,7 +172,7 @@ void main() async {
         );
 
         var serverStarted = false;
-        for (int retries = 0; retries < 10; retries++) {
+        for (int retries = 0; retries < 15; retries++) {
```

Comment:
Why all the retries and why the increase?Ø

Reply:
Due to `dart run` taking a bit more time to run than the precompiled kernel snapshot before, there's just a little bit of delay added which breaks this test. Ideally, we should have a better system to wait for the server to be ready. This was a hack to get the CI green